legal-api/migrations/versions/004f7558063f_second.py

```diff
@@ -32,9 +32,11 @@ def upgrade():
     sa.Column('change_filing_id', sa.Integer(), nullable=True),
     sa.Column('office_id', sa.Integer(), nullable=True),
     sa.Column('version', sa.Integer(), nullable=False),
+    sa.Column('alternate_name_id', sa.Integer(), nullable=True),
```

Comment:
you'll need to probably do all the same upgrade/downgrade updates you did in the alembic script for  `addresses` table to the `addresses_history` table

Reply:
that would be included in #19429 i think